sensor/common/centralclient/client_test.go

```diff
@@ -28,13 +28,22 @@ import (
 const (
 	endpoint = "localhost:8000"
 
-	// Receiving trust info examples from a running cluster:
-	// roxcurl /v1/tls-challenge?"challengeToken=h83_PGhSqS8OAvplb8asYMfPHy1JhVVMKcajYyKmrIU="
+	// 1. Deploy a new StackRox instance
+	// 2. Create and generate a new CA and replace it in the additional-ca.yaml:
+	// $ openssl genrsa -des3 -out myCA.key 2048
+	// $ openssl req -x509 -new -nodes -key myCA.key -sha256 -days 1825 -out myCA.pem -days 100000 -subj '/CN=Root LoadBalancer Certificate Authority'
```

Comment:
Shall we set a reminder in our calendars for 1st of September 2296, so that we don't forget to renew? 😉